src/amp/components/Ad.tsx

```diff
@@ -4,6 +4,7 @@ import { adJson, stringify } from '@root/src/amp/lib/ad-json';
 
 // Largest size first
 const inlineSizes = [
+	{ width: 320, height: 480 }, // Picnic story
 	{ width: 300, height: 250 }, // MPU
```

Comment:
Late comment, but I think we could change the order to start with MPU.

Reply:
I have revived this branch since it was reverted, so I can make the change. What's your reasoning behind this?

Reply:
As we take the first `width` × `height` combination below, I think starting with the safest, most popular size is better. This does mean that we need to start using the `data-override-width` and `data-override-height` attributes, otherwise multi-sized cannot be bigger than the original size.

https://github.com/guardian/dotcom-rendering/blob/b111a393badd668b3ccc32021a7898c880282c07/src/amp/components/Ad.tsx#L100

The documentation is [shockingly poor][1], but maybe that’s the incantation that will solve it?

[1]: https://github.com/ampproject/amphtml/blob/main/extensions/amp-ad-network-doubleclick-impl/multi-size.md

Reply:
Hmm, I had a go with MPU as the first size and using this
```
data-override-width={320}
data-override-height={480}
```
but it has lead to some unexpected results...

<img width="350" alt="Screenshot 2021-06-03 at 12 07 01" src="https://user-images.githubusercontent.com/19875457/120635385-52cef280-c464-11eb-8e1b-5ce7c6170e63.png">